scripts/create_compdb.py

```diff
@@ -23,145 +23,16 @@
 SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 """
 
-import json
-import re
 import subprocess
-import sys
-from pathlib import Path
 
 import scripts_utils
 
-scripts_utils.chdir_repo_root()
-directory = Path.cwd()
 
-# We use the `BAZEL` environment variable if present. If not, then we try to
-# use `bazelisk` and then `bazel`.
-bazel = scripts_utils.locate_bazel()
+def main() -> None:
+    scripts_utils.chdir_repo_root()
+    bazel = scripts_utils.locate_bazel()
+    subprocess.run([bazel, "run", "@hedron_compile_commands//:refresh_all"])
```

Comment:
Should we hide the output from this without a flag to avoid users being confused by it printing out error messages that aren't actually errors?

Reply:
The output does include messages that build errors might be spurious, and I'm wary of suppressing output that *could* be errors (or even just actionable warnings). Are you sure you want to suppress all error messages?